lib/main.ts

```diff
@@ -380,7 +388,9 @@ export async function viewConfession(
     console.log(`Publishing message...`);
     const published_message = await web.chat.postMessage({
       channel: confessions_channel,
-      text: sanitize(`*${record.id}*: ${record.text}`),
+      text: sanitize(
+        `*${record.id}*: ${tw_text == "" ? record.text : tw_text}`
```

Comment:
```suggestion
        `*${record.id}*: ${tw_text ?? record.text}`
```

Reply:
Add something that indicates it's a TW, like `TW: blah blah, open thread to view confession`